packages/scene-composer/src/utils/entityModelUtils/sceneUtils.ts

```diff
@@ -97,6 +99,46 @@ export const isDynamicScene = (document?: ISceneDocumentInternal): boolean => {
   return !isEmpty(document?.properties?.[KnownSceneProperty.SceneRootEntityId]);
 };
 
+export const useSceneRootEntityId = (document?: ISceneDocumentInternal): string => {
+  const sceneRootEntityId = document?.properties?.[KnownSceneProperty.SceneRootEntityId];
+  return sceneRootEntityId;
+};
+
+export const saveSceneNodes = async (
```

Comment:
We still have coverage on this because it's used in even more places now, but might be worth adding a unit test for this function now that's it's broken out and more testable.